cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/inmemory/InMemoryQueueAdmin.java

```diff
@@ -79,6 +79,11 @@ public void dropAll() throws Exception {
   }
 
   @Override
+  public void dropAllInNamespace(String namespaceId) throws Exception {
+    queueService.resetQueuesWithPrefix("queue:///" + namespaceId);
```

Comment:
should there be a slash after the namespaceId, so that other queues with same prefix are not dropped?